### ISSUE TITLE
Fine tuning

### ISSUE THREAD

**Issue by gauravgund** [swapnil-ahlawat/Document_Layout_Detection-MonkAI#1]
Hi,

Did you try fine tuning the page segmentation using yolov3 or you have to begin training from scratch?

**Comment by swapnil-ahlawat** [comment]
YOLOv3-Document_Analysis.ipynb notebook performs document layout analysis using pretrained yolov3. There is also a section in the notebook where you can directly use the trained weights and get the result.

**Comment by gauravgund** [comment]
How to fine-tune the model for custom datasets for better accuracy?


Best Regards

**Comment by swapnil-ahlawat** [comment]
I would request you to go through the Tutorial Blog that is there in the README.md. It covers how to train on custom datasets. In the notebook, you can find how you can load trained weights in your model. You can then train it on your custom dataset for fine-tuning.
